src/pages/transactiondetail/index.tsx

```diff
@@ -0,0 +1,558 @@
+            <div className="flex flex-col gap-y-1">
+              <div className="text-white-700">{fixedTitle[4]}</div>
+              <div>
+                <div className="flex flex-row gap-x-2.5 items-center">
+                  <CopyableText
+                    label={truncateTextFromMiddle(address.from)}
+                    onTextClick={setCopyFromAddress}
+                    textClicked={copyFromAddress}
+                  />
+                  {copyFromAddress ? (
+                    <div>
+                      <GreenTickIcon />
+                    </div>
+                  ) : (
+                    <FiCopy className="text-white-50 h-[22px]" />
+                  )}
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="grow">
+            <div className="flex flex-col gap-y-1">
+              <div className="text-white-700">{fixedTitle[5]}</div>
+              <div>
+                <div className="flex flex-row gap-x-2.5 items-center">
+                  <CopyableText
+                    label={truncateTextFromMiddle(address.to)}
+                    onTextClick={setCopyToAddress}
+                    textClicked={copyToAddress}
+                  />
+                  {copyToAddress ? (
+                    <div>
+                      <GreenTickIcon />
+                    </div>
+                  ) : (
+                    <FiCopy className="text-white-50 h-[22px]" />
+                  )}
+                </div>
+              </div>
+            </div>
+          </div>
+          <div>
+            <div className="flex flex-col gap-y-1">
+              <div className="text-white-700">{fixedTitle[6]}</div>
+              <div>
+                <div className="flex flex-row gap-x-2.5 items-center">
+                  <CopyableText
+                    label={truncateTextFromMiddle(address.contractAddress)}
+                    onTextClick={setCopyContractAddress}
+                    textClicked={copyContractAddress}
+                  />
+                  {copyContractAddress ? (
+                    <div>
+                      <GreenTickIcon />
+                    </div>
+                  ) : (
+                    <FiCopy className="text-white-50 h-[22px]" />
+                  )}
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+interface TransactionDetailSegmentTwoProps {
+  gasPrice: {
+    value: string;
+    symbol: string;
+  };
+  gasLimit: string;
+  gasUsed: string;
+  gasUsedPercentage: string;
+  from: string;
+  to: string;
+  forToken: {
+    value: string;
+    symbol: string;
+  };
+  type: string;
+  hex: string;
+}
+
+function TransactionDetailSegmentTwo({
+  gasPrice,
+  gasLimit,
+  gasUsed,
+  gasUsedPercentage,
+  from,
+  to,
+  forToken,
+  type,
+  hex,
+}: TransactionDetailSegmentTwoProps) {
+  const [hexClick, setHexClick] = useState(false);
```

Comment:
let's call the state `isRawInputExpanded`

Reply:
renamed to isRawInputExpanded